Controllers/Admin/StudentController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+namespace Controllers\Admin;
+
+use Controllers\BaseController;
+use Models\StudentModel;
+use View\StudentView;
+
+/**
+ * @package Controllers\Admin
+ */
+class StudentController extends BaseController
+{
+    /**
+     * @var StudentModel
```

Comment:
phpdocs for strict type property declaration are redundant. please remove such phpdocs